index.html

```diff
@@ -9,7 +9,7 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" rel="stylesheet">
-    <title>damdafayton Portfolio</title>
+    <title>damdafayton Portfolio Page</title>
 </head>
 
 <body>
```

Comment:
- [ ] [REQUIRED] I think the toolbar(header) should be enclosed with a `<header>` tag but you have enclosed yours with a `<nav> `tag. PLease fix that as giving it a `<header>` tag will be important for accessibility. Please also fix the hamburger button to  be enclosed in a `<nav>` tag instead of a `<div>`.
- [ ] [REQUIRED] I think that the headline section right after the header should be enclosed in a `<section>` tag. You have enclosed yours in an article tag. The section tag defines sections in a document, such as chapters, headers, footers, or any other sections of the document. ... The article tag specifies independent, self-contained content. 
- [ ] [REQUIRED] On the subtitle on line 26 starting with ''I'm a software developer'', I think you should use a header tag `<h2>,<h3>, <h4>, <h5>` or `<h6>` instead of a `<span>` tag.
- [ ] [REQUIRED] I think the paragraph on line 27 starting with "I can help..." should be enclosed in  a <p> tag instead of a section tag.
- [ ] [REQUIRED] Please also enclose the list of icons in `<ul>` tag to group them, and `<li>` per icon, on each icon that has a link (`<a>` tag)
- [ ] [REQUIRED] Please also as required in the project requirements, you must stick to the design as much as possible (e.g. font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122).

### Original Design
![port1](https://user-images.githubusercontent.com/63964646/148264324-f4e6a9cf-8d5d-4f26-b856-c9cf9fc3cea2.PNG)

### Your Design
![port2](https://user-images.githubusercontent.com/63964646/148264343-444e0c33-6fb6-4295-9f09-1ccb517a9dea.PNG)